decidim-elections/app/models/decidim/elections/election.rb

```diff
@@ -78,13 +78,18 @@ def minimum_answers?
         questions.all? { |question| question.answers.size > 1 }
       end
 
-      # Public: Checks if the election results are published and election finished
-      #
-      # Returns a boolean.
+      # # Public: Checks if the election results are published and election finished
```

Comment:
Mmm, there is any reason to double comment this? 😄

Reply:
Nope. 😅